Flank/flank
# test: [IT] Add test cases

Fixes #1313 

This PR introduces a couple of new features
1. New `integrationTests` task in `integration_test` module
1. Few test cases for IT (sanity robo, multiple apks, multiple devices, filtering tests, ignoring failed, cancelling)
1. IT test code -- it basically compares output printed to stdout with expected (`resources/compare` files)

## Test Plan
> How do we know the code works?

1. Run `./gradlew build` -- to ensure you have flank.jar
1. Run `./gradlew integrationTests`
1. All tests should finish without error (note: it may take up to 15 min to finish)
